API/ev3_sensor.c

```diff
@@ -406,9 +409,17 @@ int ReadSensor(int sensorPort)
 */
 int SetSensorMode(int sensorPort, int name)
```

Comment:
I'm reviewing this comment now because I'm working on the pull request to split ev3_sensors.c (with the very appreciated help of @JakubVanek).

I think that the function could be useful to change the sensor mode after the first call to `SetAllSensorMode`, without specifying the mode for each sensors. This is useful if, for example, you have two color sensors and you want to read light and color from each one, and you don't want to keep track of the other sensor mode in some variables. Am I missing another way to achieve the same result?